### ISSUE TITLE
Hyperparameter setting

### ISSUE THREAD

**Issue by TangYJ1229** [ChenHongruixuan/BRIGHT#11]
Could you please clarify whether the number of iterations for each model is 50,000 as mentioned in the paper or 80,000 as indicated in the code? Also, is the crop size set to 640?

**Comment by ChenHongruixuan** [comment]
Hi @TangYJ1229 ,

Please refer to our README file to see the specific hyperparameters. The default parameters in the python script is just placeholder. 

You can refer to the training script for getting the weight of the loss function.

Best,
